### ISSUE TITLE
[task] Contention Mapping | Review the historical data set we got from VBA

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#48373]
## Value Statement
**_As a_** team 
**_I want to_** review data we got from the VBA
**_So that_** we can determine which codes from that data are in the updated mapping list.

---

## Tasks
- [ ] _Filter down to clmant_txt that are exact matches to 800 dropdown list_
- [ ] _Of those rows, aggregate the classification codes we see and total counts of each_
- [ ] _Identify which of the codes are in the updated 125 list_

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by saderagsdale** [comment]
@kylesoskin added a ticket for the first data analysis request that you completed for Emily. If it's done you can move it to closed. Just wanted to capture it.

**Comment by kylesoskin** [comment]
I was able to sift through the data, and there is some meaningful things I think that have been found at least for a start or to look into. For example

```
CONGESTIVE HEART FAILURE:
  codes:
    '1830': 303
    '7005': 8979
```

This means, in all those records, records with the CNTNTN_CLSFCN_TXT matching CONGESTIVE HEART FAILURE , have 303 instances of the CNTNTN_CLSFCN_ID=1830  (which makes sense, this is the OLD code and what we [see in the drop down and are currently passing along for this contention. ](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/disability-benefits/all-claims/content/disabilityLabels.js#L184)However the more interesting part is that 8979 records have CNTNTN_CLSFCN_ID=7005 . This indicates that 7005 is probably the "NEW" contention code for CONGESTIVE HEART FAILURE, replacing 1830
we can see this with others as well
```
HYPERTENSION:
  codes:
    '3460': 12930 // OLD
    '7101': 8979 // NEW
```
However, MOST contention codes only have 1 code, and it is the same as the current code in the drop down. For example
IRRITABLE BOWEL SYNDROME:
```
  codes:
    '3570': 9624
IRRITABLE COLON:
  codes:
    '3580': 228
```
So we need to figure out what that indicates. My guess is that they no longer have a category of IRRITABLE BOWEL SYNDROME and every one submitted as that may get manually changed to something else, so we only see the old code associated with that. For example, they may change IRRITABLE BOWEL SYNDROME-> Digestive Issue , and that type of manual change is not able to be captured easily in this data.
In summary, there is some info we can get form this data, but it is not the end-all-be-all or golden ticket we had hoped due to situations like the one I just mentioned, manual edits, and data inconsistencies. A human manually looking at each of the 800 contention texts from the drop down and assigning a code in the spreadsheet I think is still going to be our best bet.

**Comment by kylesoskin** [comment]
This has been reparsed and re-evaluated. Results sent to Emily. Waiting on validation, but I think the last checkbox of "Identify which of the codes are in the updated 125 list" is not going to be ours to do/part of this ticket. So will close after emily says she is good/done with the data provided.
